helm/vitess/templates/NOTES.txt

```diff
@@ -8,6 +8,6 @@ To access administrative web pages, start a proxy with:
 
 Then use the following URLs:
 
-vtctld: {{$proxyURL}}/services/vtctld:web/app/
-vtgate: {{$proxyURL}}/services/vtgate-{{$cell}}:web/
-
+      vtctld: {{$proxyURL}}/services/vtctld:web/app/
+      vtgate: {{$proxyURL}}/services/vtgate-{{$cell}}:web/
+{{ if $.Values.orchestrator.enabled }}orchestrator: {{$proxyURL}}/services/orchestrator:web/{{ end }}
```

Comment:
Should this be indented to match the two above?

Reply:
The spacing right-aligns to the colon now, which I think works the best.